core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -531,6 +510,49 @@ class TeamPolicyInternal<Kokkos::Cuda, Properties...>
     return *this;
   }
 #endif
+
+  template <class ClosureType, class FunctorType, class BlockSizeCallable>
+  int internal_team_size_common(const FunctorType& f,
+                                BlockSizeCallable&& block_size_callable) const {
+    using closure_type = ClosureType;
+    using functor_value_traits =
+        Impl::FunctorValueTraits<FunctorType, typename traits::work_tag>;
+
+    cudaFuncAttributes attr =
+        CudaParallelLaunch<closure_type, typename traits::launch_bounds>::
+            get_cuda_func_attributes();
+    const int block_size =
+        block_size_callable(space().impl_internal_space_instance(), attr, f,
```

Comment:
(pedantic nonsense, please ignore, but in case someone would learn something:) This should *very technically* be `std::forward<BlockSizeCallable>(block_size_callable)(...)` since this is the last use of the forwarding reference `block_size_callable`.  Don't change this here; it's totally fine, but maybe it will help in a future scenario where it would actually matter.